src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -487,7 +491,71 @@ private void updateBundleAndRecipientListWithResponses(
         bundle.put(question, responses);
         recipientList.put(question.getId(), recipients);
     }
-    
+
+    /**
+     * Removes instructors who are not displayed to students from the
+     * {@link FeedbackQuestionBundle}
+     * 
+     * @param bundle
+     *            {@link FeedbackQuestionsBundle} containing questions for the
+     *            session
+     */
+    private void filterOutHiddenInstructors(FeedbackSessionQuestionsBundle bundle) {
+
+        Set<String> hiddenInstructorEmails = getHiddenInstrutorEmails(bundle.getFeedbackSession().getCourseId());
```

Comment:
Yes. First, we can get a list of feedback questions with instructor recipient type. If the list is not empty, perform database query to get instructors who are not displayed to students. After that, remove the instructors from the list of feedback questions.